e2e/playwright.config.js

```diff
@@ -21,7 +21,11 @@ export default defineConfig({
   // Opt out of parallel tests on CI.
   workers: process.env.CI ? 1 : undefined,
   // Use 'blob' for CI to allow merging of reports. See https://playwright.dev/docs/test-reporters
-  reporter: process.env.CI ? 'blob' : 'html',
+  reporter: process.env.CI
+  ? [['blob']]  :
+    // Never open the HTML report to prevent terminal from hanging on test failure
+    [['html', { open: 'never' }],
+  ],
```

Comment:
We want to use this in playwright config, and not set a Dockerfile env, so that local native `make e2e-test-native` doesn't hang